slic3r.pl

```diff
@@ -6,7 +6,7 @@
 BEGIN {
     use FindBin;
     use lib "$FindBin::Bin/lib";
-    use local::lib '--no-create', "$FindBin::Bin/local-lib";
```

Comment:
@lordofhyphens  Sorry, this file I always forget to revert it back :) . I won't leave it till the end. I do that change when I code on ubuntu 14.04. I tried to update the perl modules but I couldn't.